python/ray/data/tests/test_consumption.py

```diff
@@ -1413,12 +1413,6 @@ def test_unsupported_pyarrow_versions_check_disabled(
     except ImportError as e:
         pytest.fail(f"_check_pyarrow_version failed unexpectedly: {e}")
 
-    # Test read_parquet.
```

Comment:
Pyarrow 5 does not support pickling the Parquet reader class. Given we do not support Pyarrow 5, remove the test code here. Already verified for Pyarrow 6+, it's not an issue.

Reply:
should we just remove the PyArrow 5 CI?

Reply:
We already removed. CI only tests 6 and 12. This test manually install 5.